### PR TITLE
General Doc Fixes for azure-core, azure-core-http-netty, and azure-search-documents

### LINKED ISSUE #14400: Azure Cognitive Search samples and Reademe issues
1.
Section [Link]( https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/search/azure-search-documents#authenticate-the-client):
image
Suggestion:
Delete the `and`.
2.
Section [Link](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/search/azure-search-documents/src/samples/java/com/azure/search/documents/FieldBuilderExample.java#L34):
image
image
Reason:
Invalid annotation, please check it. 

@jongio for notification

### PR DESCRIPTION
**Pull request by alzimmermsft**
Fixes #14400, #14401, and #14406

General fixes to documentation for `azure-core`, `azure-core-http-netty`, and `azure-search-documents`.

### CODE REVIEW

File: sdk/core/azure-core-http-netty/README.md [hunk 1]
```diff
@@ -53,16 +53,16 @@ HttpClient client = new NettyAsyncHttpClientBuilder()
...
 locate the root issue. View the [logging][logging] wiki for guidance about enabling logging.
 
 ## Next steps
 
 ## Contributing
 
-If you would like to become an active contributor to this project please follow the instructions provided in [Microsoft
-Azure Projects Contribution Guidelines](http://azure.github.io/guidelines.html).
+If you would like to become an active contributor to this project please follow the instructions provided in 
+[MicrosoftAzure Projects Contribution Guidelines](http://azure.github.io/guidelines.html).
```
Comment (conniey) [inline sdk/core/azure-core-http-netty/README.md:65]: Should this have a space between the words?

File: sdk/search/azure-search-documents/README.md [hunk 2]
```diff
@@ -70,40 +62,37 @@ az search service create --name <mysearch> --resource-group <mysearch-rg> --sku
 
 In order to interact with the Azure Cognitive Search service you'll need to create an instance of the Search Client class. 
 To make this possible you will need, 
-1. [URL endpoint](https://docs.microsoft.com/en-us/azure/search/search-create-service-portal#get-a-key-and-url-endpoint) and
+1. [URL endpoint](https://docs.microsoft.com/en-us/azure/search/search-create-service-portal#get-a-key-and-url-endpoint)
```
Comment (conniey) [inline sdk/search/azure-search-documents/README.md:65]: Remove en-us so a user's locale is used.

File: sdk/search/azure-search-documents/README.md [hunk 3]
```diff
@@ -206,11 +194,9 @@ for (SearchResult searchResult: searchResultsIterable) {
...
 ## Key concepts
 
-An Azure Cognitive Search service contains one or more indexes that provide
-persistent storage of searchable data in the form of JSON documents.  _(If
-you're brand new to search, you can make a very rough analogy between
-indexes and database tables.)_  The Azure.Search.Documents client library
-exposes operations on these resources through two main client types.
+An Azure Cognitive Search service contains one or more indexes that provide persistent storage of searchable data in 
+the form of JSON documents. _(If you're new to search, you can make a very rough analogy between indexes and database 
+tables.)_ The Azure.Search.Documents client library exposes operations on these resources through two main client types.
```
Comment (conniey) [inline sdk/search/azure-search-documents/README.md:199]: Azure.Search.Documents? It looks .NET ish.

File: sdk/core/azure-core-http-netty/README.md (hunk 1 above)
Comment (alzimmermsft) [reply sdk/core/azure-core-http-netty/README.md:65]: Yes, good catch.

File: sdk/search/azure-search-documents/README.md (hunk 3 above)
Comment (alzimmermsft) [reply sdk/search/azure-search-documents/README.md:199]: I'll take a good bet that this was copied from .NET.

**Pull request merged**